src/shared/utils/upgradeForm.ts

```diff
@@ -50,7 +47,7 @@ export function extractSeats({
 
   // if their on trial their seat count is around 1000 so this resets the
   // value to the minium value they would be going on if sentry or pro
-  if (trialStatus === TrialStatuses.ONGOING && value === Plans.USERS_TRIAL) {
+  if (trialStatus === TrialStatuses.ONGOING && isTrialPlan) {
```

Comment:
Here's one of the references we were still relying on the plan name for; changed to now use isTrialPlan